2018-december-retreat/ontario-coop-requirements.md

```diff
@@ -0,0 +1,429 @@
+# Ontario Co-op Requirements
```

Comment:
What's the source of this doc? Can we link it? (Sorry, tried searching some key phrases, but can't find it online)

Reply:
It is linked from the retreat doc, this was the research doc we compiled which lives as a hackmd: https://hackmd.io/GNMLlqWpQe2LREf6umqPhw

Reply:
Do we want to link from a hackpad?

Reply:
Oh, @patcon -- it's the one I linked from above. I think I may misunderstand what you meant by source, though much comes from the Appendicies and Reference material linked throughout

Reply:
heh yeah, thought it was from elsewhere and just reformatted to markdown. Sounds like it's an assemblage though :) 🙌

Reply:
```suggestion
---
source: https://hackmd.io/GNMLlqWpQe2LREf6umqPhw
---
# Ontario Co-op Requirements
```

Thoughts on this style? I know we shouldn't expect them to change in hackmd anymore, but feels nice to keep track of where it was?

Reply:
I'm fine with following YAML conventions, though given my difficulty interpreting the word "source" I'd say it is a little ambiguous...?

tbh I don't think linking to hackmds as a "source" adds much value over time--anticipating they eventually dissolve. I think metadata that provides descriptive context will age better and not wind up as clutter (e.g., date created, authors, event short description or purpose).

Reply:
Why do you see a specific link as useful?

Reply:
I definitely don't think there is value in persisting hackmd sources that are meant to be disposable in the first place. If anything, it makes the reader more confused. I'd even go to the hackmd and delete the text, and replace it with a link to the github copy.

Reply:
I think deleting them, and linking github is best for now, if we don't want to use the hackmd docs after creating our PRs.

Reason I was leaning toward tracking it: At one point, I thought that a nightly task might be nice, that could gather changes in hackmds after the fact, and poof open new PRs for any additional changes. Tracking the source url seemed helpful if we wanted laypersons to be able to participate in notes clean-up and review. The idea was that hackmd was friendlier for external participants than our GitHub process. This was also related to https://github.com/hyphacoop/worker-coop-scripts/issues/1.

But that's nonexistent, so happy to revisit later if/when there's a need :)

Reply:
Okay, then I'll merge as-is